code/modules/mob/living/simple_animal/hostile/mining/gutlunch.dm

```diff
@@ -133,3 +133,34 @@
 	if(.)
 		udder.reagents.clear_reagents()
 		regenerate_icons()
+
+/mob/living/simple_animal/hostile/asteroid/gutlunch/grublunch
+	name = "grublunch"
+	wanted_objects = list() //They don't eat.
+	gold_core_spawnable = CHEM_MOB_SPAWN_INVALID
+	var/growth = 0
+
+//Baby gutlunch
+/mob/living/simple_animal/hostile/asteroid/gutlunch/grublunch/Initialize()
+	. = ..()
+	add_atom_colour("#9E9E9E", FIXED_COLOUR_PRIORITY) //Somewhat hidden
+	resize = 0.45
+	update_transform()
+
+/mob/living/simple_animal/hostile/asteroid/gutlunch/grublunch/Life()
+	..()
+	growth++
+	if(growth > 50) //originally used a timer for this but was more problem that it's worth.
+		growUp()
+
+/mob/living/simple_animal/hostile/asteroid/gutlunch/grublunch/proc/growUp()
+	var/mob/living/L
+	if(prob(45))
+		L = new /mob/living/simple_animal/hostile/asteroid/gutlunch/gubbuck(loc)
+	else
+		L = new /mob/living/simple_animal/hostile/asteroid/gutlunch/guthen(loc)
+	mind?.transfer_to(L)
+	L.faction = faction.Copy()
+	L.setDir(dir)
+	visible_message("<span class='notice'>[src] grows up into [L].</span>")
+	qdel(src)
```

Comment:
Whoever wrote this on TG didn't know what they were doing.

They hard referenced the old mobs factions and called `Destroy` directly---OUCH. Never call `Destroy` directly.